src/utils.js

```diff
@@ -158,3 +158,16 @@ export function sorter ($sort) {
     return comparator;
   };
 }
+
+export function makeUrl (path, app = {}) {
+  const get = typeof app.get === 'function' ? app.get : () => {};
```

Comment:
Pretty sure it works. I can try with a proper app instance instead of a mock one.